config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/AnnotatedEntity.java

```diff
@@ -96,6 +101,18 @@ public boolean isExcluded() {
         return entity instanceof AnnotableEntity && ((AnnotableEntity) entity).isExcluded();
     }
 
+    public boolean isL7Template() {
+        boolean l7Template = false;
+        switch (entityType) {
+            case EntityTypes.ENCAPSULATED_ASSERTION_TYPE:
+                l7Template = Boolean.valueOf(String.valueOf(((Encass) entity).getProperties().get(L7_TEMPLATE)));
+                break;
+            case EntityTypes.SERVICE_TYPE:
+                break;
+        }
```

Comment:
We don't need switch case here. Just 
`return entityType == EntityTypes.ENCAPSULATED_ASSERTION_TYPE && Boolean.valueOf(String.valueOf(((Encass) entity).getProperties().get(L7_TEMPLATE)));`

Reply:
we have to do it for service too, will add that code in another PR.
L7Template will be true for service, if service contains assertion - L7p:ApiPortalManagedServiceAssertion